packages/api-clients/package.json

```diff
@@ -10,6 +10,7 @@
   },
   "scripts": {
     "lint": "eslint . --ext .ts,.tsx",
+    "lint:openapi": "spectral lint -D ./open-api/!\\(selfcareV2ClientApi.yml\\|authorizationManagementApi.yaml\\|notifierApi.yml\\) --ruleset .spectral.yaml",
```

Comment:
Seeing how it works now, considering also the fact that it's not the default, and you need to change the `pr` workflow to get the spectral config file because it's not in the root, then maybe I would prefer the other option I suggested here: https://github.com/pagopa/interop-be-monorepo/pull/1719#discussion_r2137847022

I'd put the spectral file in the root and add a turbo command that runs `lint:openapi` on subpackages that have it (same as lint). In this way we also align with the behaviour of `lint` command and eslint config (that is in the root)